contracts/RootChain.sol

```diff
@@ -111,6 +111,19 @@ contract RootChain {
         uint256 _spendingTxSignatures,
         uint256 _spendingTxConfirmationSignatures
     ) public {
+		uint256 eUtxoPos = _encodedSpendingTx.getUtxoPos(_exitingUtxoPosition);
+
+        uint256 _txindex  = (_spendingTxPosition % 1000000000) / 10000;
+        bytes32 root = plasmaChain[_spendingTxPosition / 1000000000].root; //need to make plasma chain/child chain
+        var txHash = keccak256(_encodedSpendingTx);
+        var confirmationHash = keccak256(txHash, root);
+        var merkleHash = keccak256(txHash, _spendingTxSignatures);
```

Comment:
Let's get rid of `_spendingTxSignatures` here entirely and instead use `keccak256(txHash)`.